### PR TITLE
Redesign the cancel campaign progress dialog to match design

### PR DESCRIPTION
**Pull request by jjolmo**
Related to #

## Proposed Changes

* Readapted the dialog when cancelling creating a campaign to match designs

## Testing Instructions
- Create a campaign, click on " < Back" to cancel it. A dialog should appear, the dialog should be similar to this:
image

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8689976

Thank you @jjolmo for including a screenshot in the description! This is really helpful for our translators.

File: config/development.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	"logout_url": "https://wordpress.com/wp-login.php?action=logout&redirect_to=https%3A%2F%2F|subdomain|wordpress.com",
 	"facebook_api_key": "249643311490",
 	"dsp_stripe_pub_key": "pk_live_51LYYzQF53KN4RFN0oHFYHpzmHGpuUAXZ6ygDAFxM4XkRArRsET0orrofmsynkd9DhFaOGEsDZ3RC4f8mgI0zhEyN000X8i0Mqx",
-	"dsp_widget_js_src": "https://dsp.wp.com/widget.js",
+	"dsp_widget_js_src": "http://localhost:3004/widget.js",
```
Comment (sbarbosa) [inline config/development.json]: I think this change got pushed by accident, we need to change it to the correct one.

**Review by sbarbosa — COMMENTED** [review]
Hi @jjolmo, the new design looks good. 
I only found 1 difference: There is a line in the middle of the message and buttons that it is not in the designs:

Screenshot 2023-08-07 at 16 24 51

I also added a comment in the code to change a file that I think got pushed by mistake.

File: config/development.json (hunk 1 above)
Comment (jjolmo) [reply config/development.json]: whoops, I'll revert. Thanks

**Comment by jjolmo** [comment]
Done. Changes pushed to fix pr comments. Thanks @sbarbosa !!

**Review by sbarbosa — APPROVED** [review]
Nice! It worked like a charm 👍

File: client/components/blazepress-widget/index.tsx [hunk 2]
```diff
@@ -168,13 +168,18 @@ const BlazePressWidget = ( props: BlazePressPromotionProps ) => {
...
 							additionalOverlayClassNames="blazepress-widget"
 							isVisible={ showCancelDialog && showCancelButton }
 							buttons={ cancelDialogButtons }
 							onClose={ () => setShowCancelDialog( false ) }
 						>
 							<h1>{ translate( 'Are you sure you want to quit?' ) }</h1>
-							<p>{ translate( 'All progress in this session will be lost.' ) }</p>
+							<p>
+								{ translate(
+									'If quit, all of the work that has been done during this session will be lost.'
```
Comment (yuliyan) [inline client/components/blazepress-widget/index.tsx]: We've received a report from a reviewer about the missing **you** pronoun before **quit**, i.e.:
`If you quit, all of the work that has been done during this session will be lost."

Comment (jjolmo) [reply client/components/blazepress-widget/index.tsx]: I think it makes sense, I'll change it. Thanks @yuliyan

Comment (jjolmo) [reply client/components/blazepress-widget/index.tsx]: image
updated copy

**Review by yuliyan — APPROVED** [review]
Thank you for fixing the string, it looks good:

CleanShot 2023-08-10 at 13 43 28

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8689976).

**Pull request merged**